### ISSUE TITLE
The result looks weird! (running the demo file)

### ISSUE THREAD

**Issue by amena85** [matterport/Mask_RCNN#2909]
![image](https://user-images.githubusercontent.com/19568504/203987055-7ca0fd18-580f-4715-b97a-4a16967e9f60.png)

Why do I get like that when I run the demo file?

**Comment by Densus100** [comment]
it seems the issue is due to the compatibility issue of tensorflow version that you used.

**Comment by amena85** [comment]
I used tensorflow 2.11
So what should I chang in this case?

**Comment by Densus100** [comment]
yup, definitely change the tensorflow to version 1

**Comment by amena85** [comment]
I am trying to switch it to tensorflow 2. Which should be possible.

**Comment by Densus100** [comment]
nope you can't use tensorflow 2 for matterport mask rcnn

**Comment by Moe03** [comment]
> nope you can't use tensorflow 2 for matterport mask rcnn

How can I setup the demo in collab then?

I tried downgrading tensorflow version, didn't work. Also tried downgrading to python 3.6 and 3.7 and that also didn't work.

**Comment by Densus100** [comment]
> > nope you can't use tensorflow 2 for matterport mask rcnn
> 
> How can I setup the demo in collab then?
> 
> I tried downgrading tensorflow version, didn't work. Also tried downgrading to python 3.6 and 3.7 and that also didn't work.

Nope, you can't use google colab anymore to train this matterport mask r-cnn. This is due to Google Colab removed support for TensorFlow 1 in their latest release of 2022/8/11. You could see more details in https://colab.research.google.com/notebooks/relnotes.ipynb

**Comment by GilbertPan97** [comment]
> ![image](https://user-images.githubusercontent.com/19568504/203987055-7ca0fd18-580f-4715-b97a-4a16967e9f60.png)
> 
> Why do I get like that when I run the demo file?

From my experience, you can downgrade tensorflow below version 2.5, if it doesn't work, you can refer to another fork repository which supports tensorflow 2.x version: [https://github.com/leekunhee/Mask_RCNN](https://github.com/leekunhee/Mask_RCNN)

This similar issue has been raised in #2684, #2670

**Comment by huseyinbozkurtt** [comment]
My enviroment is windows10 NVIDIA GeForce RTX3060LaptopGPU
• Step 1: I create Mask_RCNN environment python version is 3.6
 conda create -n Mask_RCNN python=3.6
• Step 2:Clone the repository. First, we will clone the mask rcnn repository which has the architecture for Mask R-CNN
Download mrcnn folder from this repository: https://github.com/matterport/Mask_RCNN
• Step 3: Install the dependencies
 pip install tensorflow-gpu==1.15.0
 pip install keras==2.2.5
 pip install numpy
 pip install scipy
 pip install Pillow
 pip install cython
 pip install matplotlib
 pip install scikit-image
 pip install opencv-python
 pip install h5py==2.10.0
 pip install IPython[all]
 python setup.py install

I did all the steps but it gives me the same strange results for demo file .
